### PR TITLE
Chore: code cleanup

### PR DESCRIPTION
**Pull request by 1makedadavis**
Fixes #\<DXP 806, DXP 808\>

## Checklist

- [x] Up to date with `main`
- [x] All the tests are passing
  - [ ] Delete all resources created in tests
- [x] Prettier
- [x] Spectral Lint
- [x ] `npm run bundle` outputs nothing suspect
- [x] `npm run postman` outputs nothing suspect

### CODE REVIEW

**Comment by SidneyAllen** [comment]
@1makedadavis - can you move swap positions between TypeScript and NodeJ
Screen Shot 2022-03-15 at 10 12 42 AM
S

**Comment by SidneyAllen** [comment]
@1makedadavis - Postcard create snippet - we need the addr_xxxx in quotes
Screen Shot 2022-03-15 at 10 16 25 AM

**Comment by SidneyAllen** [comment]
@mwj8410 and @1makedadavis - I notice in Create Check we type the variable, but other code snippets don't have this.  Can we either remove this or apply it consistently for other create methods?

Screen Shot 2022-03-15 at 10 22 30 AM

**Comment by SidneyAllen** [comment]
@1makedadavis - on Verify Bank it looks like the bankData object is unnecessary - I think we should remove it

Screen Shot 2022-03-15 at 10 25 25 AM

File: resources/bank_accounts/bank_account_verify.yml [hunk 1]
```diff
@@ -92,13 +84,13 @@ post:
       label: RUBY
     - lang: Python
       source: |
-        lob.BankAccount.verify(id = 'bank_dfceb4a2a05b57e', amounts = [25, 63])
+        lob.BankAccount.verify(id = "bank_dfceb4a2a05b57e", amounts = [25, 63])
```
Comment (mwj8410) [inline resources/bank_accounts/bank_account_verify.yml:87]: should be single quotes in this file

File: resources/bank_accounts/bank_accounts.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ get:
     - lang: PHP
       source: |
         $lob->bankAccounts()->all(array(
-          'limit' => 2
+          "limit" => 2
```
Comment (mwj8410) [inline resources/bank_accounts/bank_accounts.yml:58]: swap to single quotes

File: resources/bulk_intl_verifications/bulk_intl_verifications.yml [hunk 3]
```diff
@@ -69,7 +69,7 @@ post:
       source: |
         curl https://api.lob.com/v1/bulk_intl_verifications \
           -u <YOUR_LIVE_API_KEY>: \
-          --data-raw '{
+          --data-raw "{
```
Comment (mwj8410) [inline resources/bulk_intl_verifications/bulk_intl_verifications.yml:72]: swap to single quotes

File: resources/bulk_intl_verifications/bulk_intl_verifications.yml [hunk 4]
```diff
@@ -124,12 +124,12 @@ post:
         @lob.bulk_intl_verifications.verify(
           addresses: [
             {
-              primary_line: '370 Water St',
-              secondary_line: '',
-              city: 'Summerside',
-              state: 'Prince Edward Island',
-              postal_code: 'C1N 1C4',
-              country: 'CA'
+              primary_line: "370 Water St",
```
Comment (mwj8410) [inline resources/bulk_intl_verifications/bulk_intl_verifications.yml:127]: swap to single quotes

File: resources/bulk_intl_verifications/bulk_intl_verifications.yml [hunk 5]
```diff
@@ -178,12 +178,12 @@ post:
         Lob.BulkIntlVerification.verify.verify(
         %{addresses:[
             {
-              primary_line: '370 Water St',
-              secondary_line: '',
-              city: 'Summerside',
-              state: 'Prince Edward Island',
-              postal_code: 'C1N 1C4',
-              country: 'CA'
+              primary_line: "370 Water St",
```
Comment (mwj8410) [inline resources/bulk_intl_verifications/bulk_intl_verifications.yml:181]: swap to single quotes

File: resources/bulk_us_verifications/bulk_us_verifications.yml [hunk 6]
```diff
@@ -161,15 +161,15 @@ post:
         lob.BulkUSVerification.create(
           addresses=[{
           'primary_line'='210 King Street',
-          'city'='San Francisco',
+          'city'='San Francisco",
```
Comment (mwj8410) [inline resources/bulk_us_verifications/bulk_us_verifications.yml:164]: ending quote is mismatched

File: resources/bulk_us_verifications/bulk_us_verifications.yml [hunk 7]
```diff
@@ -161,15 +161,15 @@ post:
...
           'state'='CA',
           'zip_code'='94107'
           }]
         )
       label: PYTHON
     - lang: PHP
       source: |
-        $lob->bulkUSVerifications()->verify('addresses' => array(
+        $lob->bulkUSVerifications()->verify("addresses" => array(
```
Comment (mwj8410) [inline resources/bulk_us_verifications/bulk_us_verifications.yml:172]: swap to single quotes

File: resources/cards/card.yml [hunk 8]
```diff
@@ -132,11 +132,11 @@ post:
       label: NODE
     # - lang: Python
     #   source: |
-    #     lob.Card.update('card_7a6d73c5c8457fc')
+    #     lob.Card.update("card_7a6d73c5c8457fc")
```
Comment (mwj8410) [inline resources/cards/card.yml:135]: swap to single quotes

File: resources/cards/card.yml [hunk 9]
```diff
@@ -132,11 +132,11 @@ post:
...
     #   label: PYTHON
     # - lang: PHP
     #   source: |
-    #     $lob->cards()->update('card_7a6d73c5c8457fc');
+    #     $lob->cards()->update("card_7a6d73c5c8457fc");
```
Comment (mwj8410) [inline resources/cards/card.yml:139]: swap to single quotes

File: resources/cards/cards.yml [hunk 10]
```diff
@@ -51,7 +51,7 @@ get:
     - lang: PHP
       source: |
         $lob->cards()->all(array(
-          'limit' => 2
+          "limit" => 2
```
Comment (mwj8410) [inline resources/cards/cards.yml:54]: swap to single quotes

File: resources/cards/cards.yml [hunk 11]
```diff
@@ -132,7 +132,7 @@ post:
     - lang: Javascript
       source: |
         Lob.cards.create({
-          description: 'Test card',
+          description: "Test card",
```
Comment (mwj8410) [inline resources/cards/cards.yml:135]: swap to single quotes

File: resources/checks/checks.yml [hunk 12]
```diff
@@ -59,7 +59,7 @@ get:
     - lang: PHP
       source: |
         $lob->checks()->all(array(
-          'limit' => 2
+          "limit" => 2
```
Comment (mwj8410) [inline resources/checks/checks.yml:62]: swap to single quotes

File: resources/checks/checks.yml [hunk 13]
```diff
@@ -246,16 +246,23 @@ post:
           -d "amount=22.50" \
           -d "memo=rent" \
           --data-urlencode "logo=https://s3-us-west-2.amazonaws.com/public.lob.com/assets/check_logo.png" \
-          --data-urlencode "check_bottom=<h1 style='padding-top:4in;'>Demo Check for {{name}}</h1>" \
+          --data-urlencode "check_bottom=<h1 style="padding-top:4in;">Demo Check for {{name}}</h1>" \
```
Comment (mwj8410) [inline resources/checks/checks.yml:249]: swap outer most double quotes to single quotes

File: resources/checks/checks.yml [hunk 14]
```diff
@@ -246,16 +246,23 @@ post:
...
       label: CURL
     - lang: typescript
       source: |
-        const checkCreate: CheckEditable = new CheckEditable({
-          to: addressId,
-          from: addressId,
-          bank_account: bankAccountId,
-          amount: 100
+        const checkCreate = new CheckEditable({
+        to: {
```
Comment (mwj8410) [inline resources/checks/checks.yml:255]: should read
```
  to: new AddressEditable({
    name: 'Harry Zhang',
    address_line1: '210 King St',
    address_line2: '# 6100',
    address_city: 'San Francisco',
    address_state: 'CA',
    address_zip: '94107',
  }),
```

File: resources/checks/checks.yml [hunk 15]
```diff
@@ -360,7 +367,7 @@ post:
 
         LobResponse<Check> response = new Check.RequestBuilder()
                 .setDescription("Demo Check")
-                .setCheckBottom("<h1 style='padding-top:4in;'>Demo Check for {{name}}</h1>")
+                .setCheckBottom("<h1 style="padding-top:4in;">Demo Check for {{name}}</h1>")
```
Comment (mwj8410) [inline resources/checks/checks.yml:370]: swap outer most double quotes to single quotes

File: resources/checks/checks.yml [hunk 16]
```diff
@@ -396,7 +403,7 @@ post:
           amount: 22.50,
           memo: "rent",
           logo: "https://s3-us-west-2.amazonaws.com/public.lob.com/assets/check_logo.png",
-          check_bottom: "<h1 style='padding-top:4in;'>Demo Check for {{name}}</h1>",
+          check_bottom: "<h1 style="padding-top:4in;">Demo Check for {{name}}</h1>",
```
Comment (mwj8410) [inline resources/checks/checks.yml:406]: swap outer most double quotes to single quotes

File: resources/letters/letters.yml [hunk 17]
```diff
@@ -65,7 +65,7 @@ get:
     - lang: PHP
       source: |
         $lob->letters()->all(array(
-          'limit' => 2
+          "limit" => 2
```
Comment (mwj8410) [inline resources/letters/letters.yml:68]: swap to single quotes

File: resources/letters/letters.yml [hunk 18]
```diff
@@ -254,19 +254,26 @@ post:
           -d "to[address_state]=CA" \
           -d "to[address_zip]=94107" \
           -d "from=adr_210a8d4b0b76d77b" \
-          --data-urlencode "file=<html style='padding-top: 3in; margin: .5in;'>HTML Letter for {{name}}</html>" \
+          --data-urlencode "file=<html style="padding-top: 3in; margin: .5in;">HTML Letter for {{name}}</html>" \
```
Comment (mwj8410) [inline resources/letters/letters.yml:257]: swap outer most double quotes to single quotes

File: resources/letters/letters.yml [hunk 19]
```diff
@@ -311,7 +318,7 @@ post:
             address_zip: "94107"
           },
           from: "adr_210a8d4b0b76d77b",
-          file: "<html style='padding-top: 3in; margin: .5in;'>HTML Letter for {{name}}</html>",
+          file: "<html style="padding-top: 3in; margin: .5in;">HTML Letter for {{name}}</html>",
```
Comment (mwj8410) [inline resources/letters/letters.yml:321]: swap outer most double quotes to single quotes

File: resources/letters/letters.yml [hunk 20]
```diff
@@ -363,7 +370,7 @@ post:
 
         LobResponse<Letter> response = new Letter.RequestBuilder()
                 .setDescription("Demo Letter")
-                .setFile("<html style='padding-top: 3in; margin: .5in;'>HTML Letter for {{name}}</html>")
+                .setFile("<html style="padding-top: 3in; margin: .5in;">HTML Letter for {{name}}</html>")
```
Comment (mwj8410) [inline resources/letters/letters.yml:373]: swap outer most double quotes to single quotes

File: resources/letters/letters.yml [hunk 21]
```diff
@@ -393,7 +400,7 @@ post:
             address_zip: "94107"
           },
           from: "adr_210a8d4b0b76d77b",
-          file: "<html style='padding-top: 3in; margin: .5in;'>HTML Letter for {{name}}</html>",
+          file: "<html style="padding-top: 3in; margin: .5in;">HTML Letter for {{name}}</html>",
```
Comment (mwj8410) [inline resources/letters/letters.yml:403]: swap outer most double quotes to single quotes

File: resources/postcards/postcards.yml [hunk 22]
```diff
@@ -65,7 +65,7 @@ get:
     - lang: PHP
       source: |
         $lob->postcards()->all(array(
-          'limit' => 2
+          "limit" => 2
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:68]: swap to single quotes

File: resources/postcards/postcards.yml [hunk 23]
```diff
@@ -248,18 +248,25 @@ post:
           -d "to[address_state]=CA" \
           -d "to[address_zip]=94107" \
           -d "from=adr_210a8d4b0b76d77b" \
-          --data-urlencode "front=<html style='padding: 1in; font-size: 50;'>Front HTML for {{name}}</html>" \
-          --data-urlencode "back=<html style='padding: 1in; font-size: 20;'>Back HTML for {{name}}</html>" \
+          --data-urlencode "front=<html style="padding: 1in; font-size: 50;">Front HTML for {{name}}</html>" \
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:251]: swap outer most double quotes to single quotes

File: resources/postcards/postcards.yml [hunk 24]
```diff
@@ -248,18 +248,25 @@ post:
...
-    - lang: typescript
+    - lang: Typescript
       source: |
         const postcardCreate = new PostcardEditable({
-          to: addressId,
-          from: addressId,
-          front:
-          "https://s3-us-west-2.amazonaws.com/public.lob.com/assets/templates/4x6_pc_template.pdf",
-          back: "https://s3-us-west-2.amazonaws.com/public.lob.com/assets/templates/4x6_pc_template.pdf"
+        to: {
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:258]: should read
```
to: new AddressEditable({
          name: 'Harry Zhang',
          address_line1: '210 King St',
          address_line2: '# 6100',
          address_city: 'San Francisco',
          address_state: 'CA',
          address_zip: '94107'
        }),

File: resources/postcards/postcards.yml [hunk 25]
```diff
@@ -281,7 +288,7 @@ post:
             address_zip: '94107'
           },
           from: 'adr_210a8d4b0b76d77b',
-          front: '<html style="padding: 1in; font-size: 50;">Front HTML for {{name}}</html>',
+          front: '<html style="padding: 1in; font-size: 50;">Front HTML for {{name}}</html>'',
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:291]: has an extra trailing single quote

File: resources/postcards/postcards.yml [hunk 26]
```diff
@@ -303,8 +310,8 @@ post:
             address_zip: "94107"
           },
           from: "adr_210a8d4b0b76d77b",
-          front: "<html style='padding: 1in; font-size: 50;'>Front HTML for {{name}}</html>",
-          back: "<html style='padding: 1in; font-size: 20;'>Back HTML for {{name}}</html>",
+          front: "<html style="padding: 1in; font-size: 50;">Front HTML for {{name}}</html>",
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:313]: swap outer most double quotes to single quotes

File: resources/postcards/postcards.yml [hunk 27]
```diff
@@ -313,7 +320,7 @@ post:
     - lang: Python
       source: |
         lob.Postcard.create(
-          description = 'Demo Postcard job',
+          description = "Demo Postcard job",
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:323]: swap to single quotes

File: resources/postcards/postcards.yml [hunk 28]
```diff
@@ -362,8 +369,8 @@ post:
                                 .setZip("94107")
                 )
                 .setFrom("adr_210a8d4b0b76d77b")
-                .setFront("<html style='padding: 1in; font-size: 50;'>Front HTML for {{name}}</html>")
-                .setBack("<html style='padding: 1in; font-size: 20;'>Back HTML for {{name}}</html>")
+                .setFront("<html style="padding: 1in; font-size: 50;">Front HTML for {{name}}</html>")
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:372]: swap outer most double quotes to single quotes

File: resources/postcards/postcards.yml [hunk 29]
```diff
@@ -382,8 +389,8 @@ post:
             address_zip: "94107"
           },
           from: "adr_210a8d4b0b76d77b",
-          front: "<html style='padding: 1in; font-size: 50;'>Front HTML for {{name}}</html>",
-          back: "<html style='padding: 1in; font-size: 20;'>Back HTML for {{name}}</html>",
+          front: "<html style="padding: 1in; font-size: 50;">Front HTML for {{name}}</html>",
```
Comment (mwj8410) [inline resources/postcards/postcards.yml:392]: swap outer most double quotes to single quotes

File: resources/templates/templates.yml [hunk 30]
```diff
@@ -139,8 +139,8 @@ post:
     - lang: Typescript
       source: |
         const templateCreate = new TemplateWritable({
-          description: "Newer Template",
-          html: "<html>Updated HTML for {{name}}</html>"
+          description: 'Newer Template',
```
Comment (mwj8410) [inline resources/templates/templates.yml:142]: swap to single quotes

File: resources/zip_lookups/zip_lookups.yml [hunk 31]
```diff
@@ -79,7 +79,7 @@ post:
     - lang: PHP
       source: |
         $lob->usZipLookups()->lookup(array(
-          'zip_code' => '94107'
+          "zip_code" => '94107'
```
Comment (mwj8410) [inline resources/zip_lookups/zip_lookups.yml:82]: swap to single quotes

**Review by mwj8410 — CHANGES_REQUESTED** [review]
I went ahead an noted which quotes need to be swapped

**Review by mwj8410 — APPROVED** [review]
Excellent!

**Pull request merged**